styles.css

```diff
@@ -124,16 +124,26 @@ li {
 	cursor: zoom-in;
 
 }
+
 .Form butto:active {
 	background-color: #7ce5ff;
-
 }
 
 .App--exit .Form button {
-  opacity: 0;
+	opacity: 0;
 }
 
 .Form input {
 	-webkit-appearance: inherit;
 	font-size: 1rem;
 }
+
+table {
```

Comment:
Use a class selector instead

Reply:
This should be fixed now. I used the ID to select it instead of introducing another class. I think in this case it makes sense, but you probably have stronger feelings here than I do.

My approach doesn’t facilitate reuse as much, but it uses the selector that is already in place anyway.